rides-and-fares/src/main/scala/org/apache/flink/training/exercises/ridesandfares/scala/RidesAndFaresExercise.scala

```diff
@@ -18,55 +18,73 @@
 
 package org.apache.flink.training.exercises.ridesandfares.scala
 
+import org.apache.flink.api.common.JobExecutionResult
+import org.apache.flink.api.common.state.{ValueState, ValueStateDescriptor}
+import org.apache.flink.configuration.Configuration
```

Comment:
unused (but may want to keep the Configuration import for the change below)
```suggestion
import org.apache.flink.configuration.Configuration
```